doc/mri.rst

```diff
@@ -14,7 +14,7 @@ Step by step instructions for using :func:`gui.coregistration`:
 .. autosummary::
    :toctree: generated/
 
-   get_mni_fiducials
+   coreg.get_mni_fiducials
```

Comment:
is this an API change @alexrockhill

Reply:
That looks like it is the API that it was before, I just corrected it because it would have been a change but instead we just imported and noqa'd to shallow reference it in coreg